SignInCheckIn/MinistryPlatform.Translation/Repositories/EventRepository.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MinistryPlatform.Translation.Models.DTO;
+using MinistryPlatform.Translation.Repositories.Interfaces;
+
+namespace MinistryPlatform.Translation.Repositories
+{
+    public class EventRepository : IEventRepository
+    {
+        private readonly IApiUserRepository _apiUserRepository;
+        private readonly IMinistryPlatformRestRepository _ministryPlatformRestRepository;
+        
+        public EventRepository(IApiUserRepository apiUserRepository,
+            IMinistryPlatformRestRepository ministryPlatformRestRepository)
+        {
+            _apiUserRepository = apiUserRepository;
+            _ministryPlatformRestRepository = ministryPlatformRestRepository;
+        }
+
+        public List<MpEventDto> GetEvents()
+        {
+            var apiUserToken = _apiUserRepository.GetToken();
+
+            var columnList = new List<string>
+            {
+                "Event_ID",
+                "Event_Title",
+                "Event_Start_Date",
+                "Event_Type_ID_Table.Event_Type",
+                "Congregation_ID_Table.Congregation_Name"
+            };
+
+            string dateOffsetSearchString = "";
+
+            // today + 6 days into the future
```

Comment:
Same comment as on [PR for US3508](https://github.com/crdschurch/crds-signin-checkin/pull/28) - can we use BETWEEN on the date, instead of OR-ing all the dates?